### PR TITLE
feat(contracts): allow pausing bridge contracts

### PR DESCRIPTION
**Pull request by kevinhalliday**
Add pausability to native bridge.

Allow pausing withdraws & claims separately

Additional:
- some file / constant name cleanup

issue: #1759

### CODE REVIEW

File: contracts/core/src/token/OmniBridgeL1.sol [hunk 1]
```diff
@@ -14,7 +16,7 @@ import { OmniBridgeNative } from "./OmniBridgeNative.sol";
  * @notice The Ethereum side of Omni's native token bridge. Partner to OmniBridgeNative, which is
  *         deployed to Omni's EVM.
  */
-contract OmniBridgeL1 is OwnableUpgradeable {
+contract OmniBridgeL1 is OwnableUpgradeable, PausableUpgradeable, OmniBridgeCommon {
```
Comment (ttarsi) [inline contracts/core/src/token/OmniBridgeL1.sol:19]: `OmniBridgeCommon` is both `OwnableUpgradeable` and `PausableUpgradeable`, so we shouldn't need to include those inheritances here as well

**Review by ttarsi — COMMENTED** [review]
Nice, only feedback is that solidity inheritance is bleh so it could be worth considering just duplicating a little code rather than have 2 layers of inheritance

**Review by ttarsi — APPROVED** [review]

File: contracts/core/src/token/OmniBridgeL1.sol (hunk 1 above)
Comment (kevinhalliday) [reply contracts/core/src/token/OmniBridgeL1.sol:19]: We don't need to but I like showing it explicitly here

Comment (kevinhalliday) [reply contracts/core/src/token/OmniBridgeL1.sol:19]: eh guess it looks nice with just one - ill think about your top level comment too

**Pull request merged**